Packs/CommonPlaybooks/ReleaseNotes/2_7_17.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### File Reputation
+
+- Added support for the File Reputation playbook to the Cortex Unified Platform.
```

Comment:
Please add platform tags (as this change is only for platform), like we did in this [PR](https://github.com/demisto/content/pull/41243/files)